app/controllers/api/admin/articles_controller.rb

```diff
@@ -5,8 +5,19 @@ class Api::Admin::ArticlesController < ApplicationController
   before_action :editor?
 
   def index
-      article = Article.where(published: false)
-      render json: article, each_serializer: Admin::Article::IndexSerializer
+    article = Article.where(published: false)
+    render json: article, each_serializer: Admin::Article::IndexSerializer
+  end
+
+  def show
+    article = Article.find(params[:id])
+    if article.published == true
+      render json: { message: 'This article was already published' }, status: 422
+    else
+      render json: article, serializer: Admin::Article::ShowSerializer
+    end
+  rescue StandardError => e
```

Comment:
this here you can extract with 

`rescue_from ActiveRecord::RecordNotFound, with: :render_active_record_error`

Reply:
How do i use it ?